test/integration/src/test/java/io/pravega/test/integration/controller/server/ControllerServiceTest.java

```diff
@@ -223,7 +223,7 @@ private static void getSegmentsForNonExistentStream(Controller controller) throw
             
             System.err.println("SUCCESS: Positions cannot be fetched for non existent stream");
```

Comment:
This is probably for a different issue, but why are we still using `System.err` here rather than slf4j log messages?